extensions/typescript-language-features/package.nls.json

```diff
@@ -74,6 +74,13 @@
 	"configuration.implicitProjectConfig.strictFunctionTypes": "Enable/disable [strict function types](https://www.typescriptlang.org/tsconfig#strictFunctionTypes) in JavaScript and TypeScript files that are not part of a project. Existing `jsconfig.json` or `tsconfig.json` files override this setting.",
 	"configuration.suggest.jsdoc.generateReturns": "Enable/disable generating `@return` annotations for JSDoc templates. Requires using TypeScript 4.2+ in the workspace.",
 	"configuration.suggest.autoImports": "Enable/disable auto import suggestions.",
+	"configuration.inlayHints.includeInlayParameterNameHints": "Enable/disable inlay hints of parameter name.",
+	"configuration.inlayHints.includeInlayFunctionParameterTypeHints": "Enable/disable inlay hints of function parameter name.",
+	"configuration.inlayHints.includeInlayVariableTypeHints": "Enable/disable inlay hints of variable type.",
+	"configuration.inlayHints.includeInlayParameterNameHintsWhenArgumentMatchesName": "Enable/disable inlay hints when argument matchs the parameter name.",
+	"configuration.inlayHints.includeInlayPropertyDeclarationTypeHints": "Enable/disable inlay hints of property declaration.",
+	"configuration.inlayHints.includeInlayFunctionLikeReturnTypeHints": "Enable/disable inlay hints of return type for function like.",
+	"configuration.inlayHints.includeInlayEnumMemberValueHints": "Enable/disable inlay hints of enum member.",
```

Comment:
```suggestion
	"configuration.inlayHints.includeInlayParameterNameHints": "Enable/disable inlay hints of parameter names.",
	"configuration.inlayHints.includeInlayFunctionParameterTypeHints": "Enable/disable inlay hints of function parameter names.",
	"configuration.inlayHints.includeInlayVariableTypeHints": "Enable/disable inlay hints of variable types.",
	"configuration.inlayHints.includeInlayParameterNameHintsWhenArgumentMatchesName": "Enable/disable inlay hints when argument matches the parameter names.",
	"configuration.inlayHints.includeInlayPropertyDeclarationTypeHints": "Enable/disable inlay hints of property declarations.",
	"configuration.inlayHints.includeInlayFunctionLikeReturnTypeHints": "Enable/disable inlay hints of return type for function signatures.",
	"configuration.inlayHints.includeInlayEnumMemberValueHints": "Enable/disable inlay hints of enum member values.",
```

Reply:
The second one says “parameter names” but the option is for parameter _types_.

Reply:
I would also still prefer to try to set `configuration.inlayHints.includeInlayParameterNameHintsWhenArgumentMatchesName` with something like

> **Suppress** parameter name hints on arguments whose text is identical to the parameter name

or something like that, then invert the value before sending it to TS, since this option does nothing unless `configuration.inlayHints.includeInlayParameterNameHints` is enabled, echoing our conversation on the TypeScript PR.

Reply:
Updated my suggestion.

Reply:
Updated and use `suppressInlayParameterNameHintsWhenArgumentMatchesName`